source/codegen/common_helpers.py

```diff
@@ -1,43 +1,14 @@
-def driver_name_to_pascal(driver_name):
```

Comment:
Note: `driver_name_to_pascal` and `driver_name_add_underscore` weren't used anywhere so I went ahead and removed them.